finance_dl/discover.py

```diff
@@ -0,0 +1,149 @@
+"""Retrieves transaction and balance information from Discover.
+Logs into the web interface, downloads an OFX file.
+
+This uses the `selenium` Python package in conjunction with `chromedriver` to
+scrape the Discover website.
+
+Configuration:
+==============
+
+The following keys may be specified as part of the configuration dict:
+
+- `credentials`: Required.  Must be a `dict` with `'username'` and `'password'`
+  keys.
+
+- `output_directory`: Required.  Must be a `str` that specifies the path on the
+  local filesystem where the output will be written.  If the directory does not
+  exist, it will be created.
+
+- `profile_dir`: Optional.  If specified, must be a `str` that specifies the
+  path to a persistent Chrome browser profile to use.  This should be a path
+  used solely for this single configuration; it should not refer to your normal
+  browser profile.  If not specified, a fresh temporary profile will be used
+  each time.  It is highly recommended to specify a `profile_dir` to avoid
+  having to manually enter a multi-factor authentication code each time.
+
+Output format:
+==============
+
+OFX file.
+
+Example:
+========
+
+    def CONFIG_discover():
+        return dict(
+            module='finance_dl.discover',
+            credentials={
+                'username': 'XXXXXX',
+                'password': 'XXXXXX',
+            },
+            output_directory=os.path.join(data_dir, 'discover'),
+            # firefox seems to work better on Discover's website
+            profile_dir=os.path.join(profile_dir, 'firefox'),
+            firefox=True,
```

Comment:
Remove comment about firefox and I think you removed the code for supporting user_agent?

Reply:
Fixed.